README.md

```diff
@@ -2,11 +2,11 @@
 
 <table><tr height="40px">
 <td width="150px"><img alt="icon" src="Icon.png" width="128px" height="128px" /></td>
-<td>This tool provides transparent encryption of files in git repositories using a symmetric key.<br />
+<td>This tool provides transparent encryption of files in Git repositories using a symmetric key.<br />
```

Comment:
I make this mistake often because I'm so use to reading and typing `git` for the CLI command. But when referring to Git the software in a sentence, it should be capitalized as any other proper noun.

For reference, see how https://git-scm.com/about is worded.